contributing/index.md

```diff
@@ -8,57 +8,100 @@ crumb: Contributing Fixes
 
-  1. Make sure you accept and fill out the contributor agreement
+  1. [Check out and build the code][12]
 
-  2. Edit the code and collect your changes in a change list (CL)
+  2. Fill in the [contributor agreement][13].
 
   3. If you've never submitted code before, you must add your (or your
      organization's in the case the contributor agreement is signed by your
-     organization) name and contact info to the [AUTHORS][9] file
+     organization) name and contact info to the [AUTHORS][9] file.
 
-  4. Upload the CL for review and add suitable reviewers.
+  4. Go to <https://webrtc.googlesource.com/new-password> and login with
+     your email account. This should be the same account as is returned by
+     git config user.email.
 
-  5. After getting feedback, modify and upload new version of your CL (repeat
-     as needed).
+  5. Follow the instructions on how to store the credentials in the
+     `.gitcookies` file in your home directory.
 
-  6. Commit your change, or ask a reviewer to do it for you if you don't have
-     access right.
+  6. Go to <https://webrtc-review.googlesource.com> and login with your
+     email account.
 
+  7. Then, run:
+     ~~~~~ bash
+     git cl creds-check  # If any errors, ask for help on discuss-webrtc
+     ~~~~~
 
-Please help us:
+You will not have to repeat the above. After all that, you're ready to upload:
 
-  * Remember to keep your change lists small and focused.
+  1. Do this:
+     ~~~~~ bash
+     # Assuming you're on the master branch:
+     git checkout -b my-work-branch
+     # Make changes, build locally, run tests locally
+     git commit -am "Changed x, and it is working"
+     git upload
+     ~~~~~
+     This will open a text editor showing all local commit messages, allowing you
+     to modify it before it becomes the CL description. Save and close the file to
+     proceed with the upload to the WebRTC [code review server][3].
 
-  * Never fix more than one bug in each CL.
+     The command will print a link like <https://webrtc-review.googlesource.com/c/src/+/53121>
+     if everything goes well.
 
-  * New features will probably require dividing your work into multiple CLs.
-    (No new features just yet!)
+  2. Click the CL link.
+
+  3. If you're not signed in, click the Sign In button in the top right and sign
+  in with your email.
+
+  4. Click Start Review and add a reviewer. See [Getting your CL Reviewed][14]
+  for how to choose reviewers.
+
+  5. Address any reviewer feedback:
+     ~~~~~ bash
+     # Make changes, build locally, run tests locally
+     git commit -am "Fixed X and Y"
+     git upload
+     ~~~~~
+
+  6. Once the reviewer LGTMs (approves) the patch, ask them to put it into the
+  commit queue.
+
+**NOTICE:** On Windows, you'll need to run the above in a Git bash shell in order
+for gclient to find the `.gitcookies` file.
+
+## Becoming a Committer
+
+After 10-20 non-trivial patches you can apply for commit rights. If you are
```

Comment:
Should we also mention how to ask for this? E.g. discuss-webrtc?

Reply:
Sure; done.